specification/network/resource-manager/Microsoft.Network/stable/2022-07-01/networkVirtualAppliance.json

```diff
@@ -1170,6 +1186,41 @@
         }
       },
       "description": "Properties of the Inbound Security Rules resource."
+    },
+    "DelegationProperties": {
+      "type": "object",
+      "properties": {
+        "serviceName": {
+          "type": "string",
+          "description": "The service name to which nva is delegated."
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState"
+        }
+      },
+      "description": "Properties of the delegation."
+    },
+    "PartnerManagedResourceProperties": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
```

Comment:
How about set format as [`arm-id`](https://azure.github.io/autorest/extensions/#x-ms-arm-id-details) for all resource ids?

Reply:
Set the format to arm-id